src/python_routing_v02/compute_nhd_routing_SingleSeg_v02.py

```diff
@@ -386,7 +427,7 @@ def main():
             jobs = []
             for twi, (tw, reach_list) in enumerate(reaches_bytw.items(), 1):
                 r = list(chain.from_iterable(reach_list))
-                param_df_sub = param_df.loc[
+                channel_initial_states_df = param_df.loc[
```

Comment:
this seems to overwrite channel_initial_states_df, which contains our initial condition data?